app/src/sharedTest/java/org/oppia/app/splash/SplashActivityTest.kt

```diff
@@ -0,0 +1,47 @@
+package org.oppia.app.splash
+
+import androidx.fragment.app.Fragment
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.intent.rule.IntentsTestRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.HomeActivity
+
+/** Tests for [SplashActivity]. */
+@RunWith(AndroidJUnit4::class)
+class SplashActivityTest {
+
+  @get:Rule
+  var mActivityRule: IntentsTestRule<SplashActivity> = IntentsTestRule(
```

Comment:
Prefer not prefixing fields with 'm', 's', etc. that AOSP code typically does. This isn't conventional for regular Java/Kotlin code, and is a bit verbose given that it's clear from context which variables are members/statics.

Reply:
corrected it.